packages/dynamic-links/android/build.gradle

```
@@ -60,16 +60,24 @@ project.ext {
 }
 
 android {
+  def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION.tokenize('.')[0].toInteger()
+  if (agpVersion >= 7) {
+    namespace 'io.invertase.firebase.dynamic-links'
```

```suggestion
    namespace 'io.invertase.firebase.dynamiclinks'
```

To match current `AndroidManifest.xml`

>     package="io.invertase.firebase.dynamiclinks">

```

* What went wrong:
Execution failed for task ':react-native-firebase_dynamic-links:generateDebugRFile'.
> A failure occurred while executing com.android.build.gradle.internal.res.GenerateLibraryRFileTask$GenerateLibRFileRunnable
   > Package 'io.invertase.firebase.dynamic-links' from AndroidManifest.xml is not a valid Java package name as 'dynamic-links' is not a valid Java identifier.

```

---

@AndreiCalazans do you want to rebase that in then I rebase+merge the whole PR, or do you want to commit it (or have me commit it) as a suggestion then squash+merge? I could go either way but so far your commits are clean + tell a story so perhaps you want to keep them clean

I usually do clean commit trails on my PRs but it is definitely not a requirement here esp. for people taking the time to help :-)

---

I had a minute so I pulled the PR and edited the first commit with this change to keep it clean, then re-pushed 🤜 🤛